src-react/__tests__/middlewares/redux-api/ungit-config.test.js

```diff
@@ -3,8 +3,9 @@ import sinon from 'sinon';
 import expect from 'expect.js';
 import configureStore from 'redux-mock-store';
 import { createStore, applyMiddleware } from 'redux';
-import { apiMiddleware, CALL_API } from 'redux-api-middleware';
+import { apiMiddleware } from 'redux-api-middleware';
 
+import { FETCH_HEADER } from '../../utils/constants';
```

Comment:
Can we require util without dots?
ex. `import { FETCH_HEADER } from 'utils/constants'`